content/pt-br/docs/concepts/policy/pid-limiting.md

```diff
@@ -0,0 +1,123 @@
+respectivamente.
+
+{{< note >}}
+Antes da versão 1.20 do Kubernetes, limitação de recursos PID com nível de node
+reservas necessárias habilitando o [recurso
+portão](/docs/reference/command-line-tools-reference/feature-gates/)
+`SupportNodePidsLimit` para funcionar.
+{{< /note >}}
+
+## Limites de PID do pod
+
+O Kubernetes permite limitar o número de processos em execução em um pod. Você
+especifique esse limite no nível do node, em vez de configurá-lo como um recurso
+limite para um pod específico. Cada node pode ter um limite de PID diferente.
+Para configurar o limite, você pode especificar o parâmetro de linha de comando `--pod-max-pids`
+para o kubelet ou defina `PodPidsLimit` no kubelet
+[arquivo de configuração](/docs/tasks/administer-cluster/kubelet-config-file/).
+
+{{< note >}}
+Antes da versão 1.20 do Kubernetes, era necessário habilitar a limitação de recursos PID para pods
+o [portão de recursos](/docs/reference/command-line-tools-reference/feature-gates/)
+`SupportPodPidsLimit` para funcionar.
+{{< /note >}}
+
+## Remoção baseada em PID
+
+Você pode configurar o kubelet para iniciar o encerramento de um pod quando ele estiver se comportando mal e consumindo uma quantidade anormal de recursos.
+Esse recurso é chamado de despejo. Você pode
+[Configurar fora da manipulação de recursos](/docs/concepts/scheduling-eviction/node-pressure-eviction/)
+para vários sinais de despejo.
+Use o sinal de remoção `pid.available` para configurar o limite para o número de PIDs usados pelo pod.
+Você pode definir políticas de despejo flexíveis e rígidos.
+Porém, mesmo com a política de despejo rígido, se o número de PIDs crescer muito rápido,
+o node ainda pode entrar em estado instável ao atingir o limite de PIDs do node.
+O valor do sinal de despejo é calculado periodicamente e NÃO impõe o limite.
+
+Limitação de PID - por pod e por node define o limite rígido.
+Assim que o limite for atingido, a carga de trabalho começará a apresentar falhas ao tentar obter um novo PID.
+Pode ou não levar ao reagendamento de um Pod,
+dependendo de como a carga de trabalho reage a essas falhas e como vivacidade e prontidão
+os probes são configurados para o pod. No entanto, se os limites forem definidos corretamente,
+você pode garantir que a carga de trabalho de outros pods e os processos do sistema não ficarão sem PIDs
+quando um Pod está se comportando mal.
+
+
+## {{% heading "whatsnext" %}}
+
+- Consulte o [Documento de aprimoramento de limitação de PID](https://github.com/kubernetes/enhancements/blob/097b4d8276bc9564e56adf72505d43ce9bc5e9e8/keps/sig-node/20190129-pid-limiting.md) para obter mais informações.
+- Para uso de conxtexto histórico, leia:
+  [Limitação de ID de processo para melhorias de estabilidade no Kubernetes 1.14](/blog/2019/04/15/process-id-limiting-for-stability-improvements-in-kubernetes-1.14/).
+- Leia [Gerenciando recursos para contêineres](/docs/concepts/configuration/manage-resources-containers/).
```

Comment:
"/docs/concepts/configuration/manage-resources-containers/" is also translated and can be updated with the /pt-br/ prefix.

Reply:
Done